### ISSUE TITLE
Step 6 Catting to suspects.txt

### ISSUE THREAD

**Issue by FiddlersCode** [makersacademy/problem-solving#9]
![image](https://cloud.githubusercontent.com/assets/7489685/24044384/fa5e90ba-0b12-11e7-9638-de18a84a33c7.png)

So getting text to cat into a file is clearly my nemesis!  As you can see, I've tried a few things to get all this info into suspects.txt but it doesn't work.

**Comment by AlexChalk** [comment]
Hi @pmuldoon86, I can see a `-B` option with no number specified, which is, I suspect, what's breaking this.

I would also try only using "vehicles" once. When you pipe `grep -A6 "L337" mystery/vehicles` to `grep -A5 "Honda"`, you're basically saying to grep, "Hey `grep`, don't search a file or directory this time round, just search the results of this earlier `grep` instead." Does that make sense?

**Comment by ilarne** [comment]
Hey @pmuldoon86, to further Alexander's suggestion, I would also suggest taking a look at the inverted comma after 6 (the search parameter being used to filter for height). I was having similar issues with the file not being created properly, and what I believe happened was that the inverted comma after 6 was being treated as a special character, rather than part of the search string, and this was disrupting the input? 

To test this, I used the 'escape', AKA backslash character, to specify that I wanted the inverted comma to be treated as part of the string. After this fix my suspects.txt file was created with no issue. Perhaps seeing it might help (see first line): 

<img width="1095" alt="screen shot 2017-03-17 at 13 30 28" src="https://cloud.githubusercontent.com/assets/22546753/24045037/a5e45044-0b15-11e7-96b0-40519f57d9ba.png">

This is the contents of the file I got after  **(spoilers below for those not past this point!!):**

<img width="986" alt="suspects screenshot" src="https://cloud.githubusercontent.com/assets/22546753/24045156/01692b24-0b16-11e7-91a9-7640d97c0e13.png">

The next step mentions only four suspects (yes I peeked), so I'm thinking this worked...hope this helps!
